Apps/Contoso.Forms.Demo/Contoso.Forms.Demo.UWP/Contoso.Forms.Demo.UWP.csproj

```diff
@@ -11,13 +11,14 @@
     <AssemblyName>Contoso.Forms.Demo.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.14393.0</TargetPlatformVersion>
-    <TargetPlatformMinVersion>10.0.10586.0</TargetPlatformMinVersion>
+    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.16299.0</TargetPlatformMinVersion>
```

Comment:
Any specific reason to bump minimum version? Creators Fall is still recent, we have older phones.

Reply:
.NET Standard 2.0 on UWP requires [the minimum version to be set to Fall Creator's Update](https://blogs.msdn.microsoft.com/dotnet/2017/10/10/announcing-uwp-support-for-net-standard-2-0/). Also Xamarin.Forms 3.0 on UWP [has a dependency on .NET Standard 2.0](https://github.com/xamarin/Xamarin.Forms/issues/2859).

I guess this is OK for *one* of our test apps.

Reply:
We don't have a non forms app that uses nuget references, the one we have is project references only. I think it would be much safer to have another blank app (just a hardcoded button in center to track a predefined event, and push/crashes enabled with no customization) that we can use to test compatibility with the arm devices we have on older versions we support.